### PR TITLE
Update anshbhimani.json

### PR DESCRIPTION
**Pull request by anshbhimani**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/anshbhimani.json [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
         "username": "AnshBhimani",
-        "email": "anshbhimani2003@gmail.com"
+        "email": "anshbhimani2003@gmail.com",
+        "twitter": "AnshBhimani"
     },
     "record": {
-        "A": ["217.174.245.249"],
-        "MX": ["hosts.is-a.dev"],
-        "TXT": "v=spf1 a mx ip4:217.174.245.249 ~all"
+        "CNAME": "anshbhimani.github.io"
```
Comment (21Z) [inline domains/anshbhimani.json:10]: 404 site not found

**Review by 21Z — CHANGES_REQUESTED** [review]

**Comment by anshbhimani** [comment]
i want to remake the website from github pages can you delete the existing one

**Comment by 21Z** [comment]
> i want to remake the website from github pages can you delete the existing one

Delete it in your fork

**Comment by anshbhimani** [comment]
Deleted in my fork

**Comment by anshbhimani** [comment]
Deleted in my fork

**Pull request closed without merging**